routes/users.js

```diff
@@ -50,4 +50,7 @@ router.patch("/rejectDiff", authenticate, authorizeRoles([SUPERUSER]), users.rej
 router.patch("/:userId", authenticate, authorizeRoles([SUPERUSER]), users.updateUser);
 router.get("/suggestedUsers/:skillId", authenticate, authorizeRoles([SUPERUSER]), users.getSuggestedUsers);
 
+// WARNING!! - One time Script/Route to do migration.
+router.post("/migrate", authenticate, authorizeRoles([SUPERUSER]), users.addGithubId);
```

Comment:
migrate represents an action/verb verbs should not be a part of the API name

Reply:
`user-data-migrator`
Is this fine?